CMakeLists.txt

```diff
@@ -51,6 +51,15 @@ if (MSVC)
   endif()
 endif()
 
+if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 14.0)
+  # Clang versions below 14 have issues with consteval in fmt: https://github.com/fmtlib/fmt/issues/2455
+  add_definitions(-DFMT_CONSTEVAL=)
```

Comment:
Please use `STREQUAL` instead of `MATCHES`, as the latter is for regular expressions. Also, prefer `add_compile_definitions()` to `add_definitions()`, as the latter is semi-deprecated.